client/account-settings/index.js

```diff
@@ -2,12 +2,12 @@
  * External dependencies
  */
 import React from 'react';
-import { combineReducers } from 'redux';
+import { combineReducers } from 'state/utils';
```

Comment:
@DanReyLop I'm still not sold on using Calypso's `combineReducers` - why do we need it? Calypso has it because they persist almost all of their state, but we persist almost none iirc.

> there's no reason to not use it.

What's the reason _to_ use it, though?